pkg/reconciler/taskrun/resources/cloudevent/cloud_event_controller_test.go

```diff
@@ -114,7 +114,7 @@ func TestSendCloudEvents(t *testing.T) {
 				SendSuccessfully: true,
 			}
 			err := SendCloudEvents(tc.taskRun, NewFakeClient(&successfulBehaviour), logger)
-			if err == nil {
+			if err != nil {
 				t.Fatalf("Unexpected error sending cloud events: %v", err)
```

Comment:
@afrittoli I'm a little confused by this fatal message and the `if` above it.  Is it supposed to detect `error`s or `nil`s?  Reads to me like it should be `if err != nil` but want to check with you before I merge it like this.

Reply:
That was a silly double-error - mistake in the code, enforced by the test :S 
Thanks for catching / fixing that!